lib/node_modules/@stdlib/math/base/special/logf/examples/index.js

```diff
@@ -18,16 +18,18 @@
 
 'use strict';
 
-var randu = require( '@stdlib/random/base/randu' );
-var roundf = require( '@stdlib/math/base/special/roundf' );
+var randu = require( '@stdlib/random/array/discrete-uniform' );
 var logf = require( './../lib' );
 
-var b;
-var x;
+var opts = {
+	'dtype': 'float32'
+};
+
+var x = randu( 100, 0, 100, opts );
+var b = randu( 100, 0, 5, opts );
+
 var i;
 
```

Comment:
```suggestion
```

The reason for this nit is that we're trying to get our examples to a state of uniformity such that we can use code translation to convert to and from ES2015+ in web docs.